Retrie/Options.hs

```diff
@@ -142,7 +143,7 @@ defaultOptions fp = Options
 -- to 'resolveOptions' to get final 'Options'.
 getOptionsParser :: FixityEnv -> IO (Parser ProtoOptions)
 getOptionsParser fEnv = do
-  dOpts <- defaultOptions <$> getCurrentDirectory
+  dOpts <- defaultOptions fEnv <$> getCurrentDirectory
   return $ buildParser dOpts { fixityEnv = fEnv }
```

Comment:
Can remove the record field assignment here since you are passing into `defaultOptions`

Reply:
I have changed `defaultOptions` to keep the previous type signature, so this record field assignment stays